lib/manageiq/release/git_mirror.rb

```diff
@@ -124,7 +144,7 @@ def upstream_branch?(branch)
       end
 
       def remote_branch?(branch)
-        !`git branch -r | grep #{branch}`.strip.empty?
+        !`git branch -r | grep "\\b#{branch}\\b"`.strip.empty?
```

Comment:
@bdunne This was an interesting bug.  In my tests I chose to mirror `ivanchuk` -> `i` but since the grep didn't check on work boundaries then `downstream/i` matched on `downstream/ivanchuk`, which was giving me some weird errors as a result.